testing/logstash/logstash.conf

```diff
@@ -0,0 +1,12 @@
+input {
+        file {
+        type => "syslog"
+        path => ["/var/log/**/*.log"]
+        }
+}
+
+output {
+        elasticsearch { hosts => ["localhost:9200"] }
```

Comment:
`logstash.conf` has `elasticsearch` input commented out now  - can it be pulled ?